src/applications/vaos/appointment-list/components/FutureAppointmentsList.jsx

```diff
@@ -11,6 +11,7 @@ import {
   selectFeatureRequests,
   selectFeaturePastAppointments,
   selectIsCernerOnlyPatient,
+  selectFeatureHomepageRefresh,
```

Comment:
I think it would be better to copy this file than modify it.

In addition to the changes you made, we'd also have to fix the header and remove the tabpanel markup. So I think it's easier to create a copy and use that in the new appointments page component you created.

Also, I don't think working on this list was part of the ticket. The AC in the ticket was to create a shell page with the dropdown. So you could delete it entirely and leave it to later tickets. If you do delete it, you could put the work you did in `ListItem` in a separate file so that Lance and I can use it in the next ticket.

Reply:
Done.